ttg/ttg/util/version.cc

```diff
@@ -1,7 +1,5 @@
-#include "../../ttg/util/version.h"
+#include "ttg/util/version.h"
 
 namespace ttg {
-  std::array<int,3> version() {
-    return {0,1,0};
-  }
-}
\ No newline at end of file
+  std::array<int, 3> version() { return {0, 1, 0}; }
```

Comment:
For the record: I'm not a fan of single-line-functions-everywhere. This change does not improve readability :/ (there are similar changes in this PR for which this is true too)

Reply:
looks like auto-clang-formatting by the git-hook ... will have a look.